shared/common-adapters/radio-button.desktop.tsx

```diff
@@ -18,45 +22,46 @@ const StyledRadio = styled.div(
     width: RADIOBUTTON_SIZE,
   },
   ({disabled, selected}) => ({
-    '&:hover': {border: (selected || !disabled) && `solid 1px ${globalColors.blue}`},
-    backgroundColor: selected ? globalColors.blue : 'inherit',
-    border: `solid 1px ${globalColors.black_20}`,
+    '&:hover': {border: (selected || !disabled) && `solid 1px ${Styles.globalColors.blue}`},
+    backgroundColor: selected ? Styles.globalColors.blue : 'inherit',
+    border: `solid 1px ${Styles.globalColors.black_20}`,
     opacity: disabled ? 0.4 : 1,
   })
 )
 
 const RadioButton = ({disabled, label, onSelect, selected, style}: Props) => (
   <div
     // @ts-ignore clash between StylesCrossPlatform and React.CSSProperties
-    style={{...styleContainer, ...(disabled ? {} : desktopStyles.clickable), ...style}}
+    style={{...styles.container, ...(disabled ? {} : Styles.desktopStyles.clickable), ...style}}
```

Comment:
is this going to cause problems when people pass in styles in `style` that conflict with `styles.container`?

Reply:
no, it overwrites it

Reply:
we also do this everywhere. it has a base style and you can override it